Code/Editor/ResourceSelectorHost.cpp

```diff
@@ -101,10 +101,10 @@ class CResourceSelectorHost
     }
 
 private:
-    typedef std::map<string, const SStaticResourceSelectorEntry*, stl::less_stricmp<string> > TTypeMap;
+    typedef std::map<AZStd::string, const SStaticResourceSelectorEntry*, stl::less_stricmp<AZStd::string> > TTypeMap;
```

Comment:
it can't its doing a case-insensitive compare

Reply:
Is this meant to be used for a path.
Because if so it would be simpler to replace the type with AZ::IO::Path and that will take care of the case-sensitive issues per platform.

Reply:
have to draw the line somewhere...

Reply:
I am good with not changing this anymore.

This feels like we shouldn't even be using this anymore in the first.

Reply:
I looked into ResourceSelectorHost-related code.  It can still be traced back to TrackView, so it's not quite ready to be removed yet until TV is updated.

ResourceSelectorHost along with the `stl::` utilities should be on the chopping block in the future.

Reply:
likely is not being used... this is using that `REGISTER_RESOURCE_SELECTOR` which is likely stripped away